docs/source/guides/snippets/grpc/README.md

```diff
@@ -0,0 +1,48 @@
+# `Serving with gRPC` snippets
```

Comment:
Honestly I'm thinking to move this to `examples`, but then it would be impossible to reference code snippets to the docs.